### PR TITLE
[ENH] Add support for HiP-CT

### PR DESCRIPTION
**Pull request by balbasty**
Resolves: #1646

This PR introduces a new `hipCT` microscopy suffix.

It does **not** make any changes to the required or optional metadata. 
To me, HiP-CT is not a quantitative modality, so I do not see any parameters that would be _required_ for downstream analysis.
Please give feedback if you feel that this is not correct and that hip-ct-specific metadata should be added. 

I am leaving it as a draft PR for now.

cc @satra @ucbpclw

### CODE REVIEW

**Comment by effigies** [comment]
This seems fine to me, and there were no objections on the discussion #1646.

@jcohenadad Do you see any problems with this?

**Comment by balbasty** [comment]
Thanks! I removed the draft mode

**Review by effigies — APPROVED** [review]

File: src/modality-specific-files/microscopy.md
Comment (effigies) [inline src/modality-specific-files/microscopy.md:92]: Small style cleanup.

```suggestion
         "SPIM",
         "hipCT",
```

**Comment by effigies** [comment]
Note that [by the rules](https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md#rules) we do need a second approving review by a community member. (It does not need to be a maintainer.)

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**
